library/src/main/java/com/joom/grip/FromConfigurator.kt

```diff
@@ -19,15 +19,19 @@ package com.joom.grip
 import com.joom.grip.mirrors.ClassMirror
 import com.joom.grip.mirrors.Type
 import java.io.File
+import java.nio.file.Path
 
 interface FromConfigurator<M, R> {
   infix fun from(classMirrorSource: ClassMirrorSource): QueryConfigurator<M, R>
+
+  @Deprecated("Deprecated.", ReplaceWith("fromPaths(files.map{ it.toPath() })"))
   infix fun from(files: Iterable<File>): QueryConfigurator<M, R>
+  infix fun fromPaths(paths: Iterable<Path>): QueryConfigurator<M, R>
```

Comment:
All overloads are named `from` to make Grip queries look like SQL queries. I believe this one should be `from` too.

Reply:
it's not possible to do, cause it will lead to declaration clash: both methods have same signature(except generic type)